{"info": {"author": "aRkadeFR", "author_email": "contact@arkade.info", "bugtrack_url": null, "classifiers": ["Framework :: Django"], "description": "UNKNOWN", "description_content_type": null, "docs_url": null, "download_url": "https://github.com/aRkadeFR/django-exceptions/tarball/v0.4", "downloads": {"last_day": -1, "last_month": -1, "last_week": -1}, "home_page": "https://github.com/aRkadeFR/django-exceptions", "keywords": null, "license": "UNKNOWN", "maintainer": null, "maintainer_email": null, "name": "django-exceptions", "package_url": "https://pypi.org/project/django-exceptions/", "platform": "UNKNOWN", "project_url": "https://pypi.org/project/django-exceptions/", "project_urls": {"Download": "https://github.com/aRkadeFR/django-exceptions/tarball/v0.4", "Homepage": "https://github.com/aRkadeFR/django-exceptions"}, "release_url": "https://pypi.org/project/django-exceptions/0.4/", "requires_dist": null, "requires_python": null, "summary": "Django app to handle exceptions in a comfortable way", "version": "0.4", "yanked": false, "html_description": "<div class=\"project-description\">\n            <p>UNKNOWN</p>\n\n          </div>"}, "last_serial": 1248674, "releases": {"0.1": [], "0.2": [], "0.3": [{"comment_text": "", "digests": {"md5": "4038b84b0ef11bf64edc47691a43a2d7", "sha256": "1ddf8c35636290d071db1ba2289e3abd0247f28340c65e7ddcde739821aa34ad"}, "downloads": -1, "filename": "django-exceptions-0.3.tar.gz", "has_sig": false, "md5_digest": "4038b84b0ef11bf64edc47691a43a2d7", "packagetype": "sdist", "python_version": "source", "requires_python": null, "size": 1517, "upload_time": "2014-10-05T19:09:02", "upload_time_iso_8601": "2014-10-05T19:09:02.069502Z", "url": "https://files.pythonhosted.org/packages/ed/58/815f73359234c70ed9cfb6644beccfa29cfa476af6e03a536308549f7911/django-exceptions-0.3.tar.gz", "yanked": false}], "0.4": [{"comment_text": "", "digests": {"md5": "dfab3124708d2cba1358538e756f95f3", "sha256": "9e0105ae3617bd31a1be2a8c3c16d74c0dc1e2c248acd9858f49935720d96e7b"}, "downloads": -1, "filename": "django-exceptions-0.4.tar.gz", "has_sig": false, "md5_digest": "dfab3124708d2cba1358538e756f95f3", "packagetype": "sdist", "python_version": "source", "requires_python": null, "size": 1546, "upload_time": "2014-10-05T19:14:06", "upload_time_iso_8601": "2014-10-05T19:14:06.643568Z", "url": "https://files.pythonhosted.org/packages/36/bb/339f038bb199895eaf1e13446e1e6ba58640da670f9dc5c2ad50eb0bd4f2/django-exceptions-0.4.tar.gz", "yanked": false}]}, "urls": [{"comment_text": "", "digests": {"md5": "dfab3124708d2cba1358538e756f95f3", "sha256": "9e0105ae3617bd31a1be2a8c3c16d74c0dc1e2c248acd9858f49935720d96e7b"}, "downloads": -1, "filename": "django-exceptions-0.4.tar.gz", "has_sig": false, "md5_digest": "dfab3124708d2cba1358538e756f95f3", "packagetype": "sdist", "python_version": "source", "requires_python": null, "size": 1546, "upload_time": "2014-10-05T19:14:06", "upload_time_iso_8601": "2014-10-05T19:14:06.643568Z", "url": "https://files.pythonhosted.org/packages/36/bb/339f038bb199895eaf1e13446e1e6ba58640da670f9dc5c2ad50eb0bd4f2/django-exceptions-0.4.tar.gz", "yanked": false}], "timestamp": "Fri May  8 00:36:17 2020"}